### PR TITLE
Polish and English Files - add missing stars in Antiphonas

### PR DESCRIPTION
**Pull request by mbab**
+AMDG+

### CODE REVIEW

File: web/www/horas/English/Sancti/11-30.txt [hunk 1]
```diff
@@ -270,7 +270,7 @@ shall not be confounded.
 $Deo gratias
 
 [Ant 2]
-Spare unto us this just man, release unto us this holy one * slay not the~
+Spare unto us * this just man, release unto us this holy one * slay not the~
```
Comment (gregordick) [inline web/www/horas/English/Sancti/11-30.txt]: Two asterisks here.

File: web/www/horas/English/Sancti/11-30.txt [hunk 2]
```diff
@@ -343,7 +343,7 @@ Christ's dear handmaid Maximilla took the body * of the Apostle and buried it wi
 O Lord, Thou hast caused them that persecuted the just to be swallowed up in hell, * but to the just Thou hast thyself shown the way on the tree of the cross.;;138
 
 [Ant 3]
-When the blessed Andrew came to~
+When the blessed Andrew * came to~
```
Comment (gregordick) [inline web/www/horas/English/Sancti/11-30.txt]: There's also an asterisk on the next line.

File: web/www/horas/English/Sancti/09-12.txt [hunk 3]
```diff
@@ -16,7 +16,7 @@ Doxology=Nat
 BMV
 
 [Ant 1]
-O Holy Mary, be thou an help to the helpless, a strength to the fearful, a~
+O Holy Mary, * be thou an help to the helpless, a strength to the fearful, a~
```
Comment (gregordick) [inline web/www/horas/English/Sancti/09-12.txt]: There's also an asterisk on the next line.

Comment (mbab) [inline web/www/horas/English/Sancti/09-12.txt]: All corected. Though star should be somewhere as in latin :-(

Comment (gregordick) [inline web/www/horas/English/Sancti/09-12.txt]: I'm happy for you to move the star to match the Latin where possible.  Would you prefer me to merge the change as it is now, or would you rather move those stars?

Comment (mbab) [inline web/www/horas/English/Sancti/09-12.txt]: Star moved.

**Comment by gregordick** [comment]
Great, thanks!

**Pull request merged**